.github/workflows/python.yml

```
@@ -215,7 +216,7 @@ jobs:
         # see https://github.com/actions/setup-python
         uses: actions/setup-python@v5
         with:
-          python-version: '>=3.8 <=3.12'  # supported version range
+          python-version: '>=3.8 <=3.13'  # supported version range
```

I suppose Python 3.8 can be dropped as it reached EOL: https://endoflife.date/python
```suggestion
          python-version: '>=3.9 <=3.13'  # supported version range
```

---

nope. not intended.